yazi-dds/src/body/hey.rs

```diff
@@ -15,7 +15,7 @@ pub struct BodyHey {
 impl BodyHey {
 	#[inline]
 	pub fn owned(peers: HashMap<u64, Peer>) -> Body<'static> {
-		Self { peers, version: BodyHi::version() }.into()
+		Self { peers, version: BodyHi::version().to_string() }.into()
```

Comment:
Please also change the `version`'s type to `&'static str` or `Cow<'static, str>`

Reply:
@sxyazi Changing type of `version` to `&'static str` gives rise to this error:
```
error[E0521]: borrowed data escapes outside of associated function
```